test-data/unit/check-classes.test

```diff
@@ -899,17 +899,17 @@ a.f.x # E: "int" has no attribute "x"
 -- -------------------------------------------------------
 
 
-[case testInvalidMultipleInheritanceFromBuiltins-skip]
+[case testInvalidMultipleInheritanceFromBuiltins]
 import typing
-class A(int, str): pass # E: Instance layout conflict in multiple inheritance
+class A(int, function): pass # E: Instance layout conflict in multiple inheritance
```

Comment:
str and int have conflicting definition of __add__, and this test is not about that.